apis/soapify-main-api.yaml

```
@@ -0,0 +1,584 @@
+  - name: Encounters
+    description: 'مدیریت ملاقات‌های پزشکی'
+  - name: STT
+    description: 'تبدیل گفتار به متن'
+  - name: NLP
+    description: 'پردازش زبان طبیعی و تولید پیش‌نویس'
+  - name: Checklist
+    description: 'چک‌لیست‌های ارزیابی'
+  - name: Outputs
+    description: 'نهایی‌سازی و خروجی‌ها'
+  - name: Integrations
+    description: 'یکپارچه‌سازی با سیستم‌های خارجی'
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)